src/Api/Middlewares/ExceptionMiddleware.cs

```diff
@@ -88,6 +89,13 @@ private async void HandleLimitExceededException(HttpContext context, Exception e
         await WriteExceptionMessageAsync(context, ex);
     }
 
+    private async void HandleInvalidOperationException(HttpContext context, Exception ex)
+    {
+        context.Response.StatusCode = StatusCodes.Status400BadRequest;
```

Comment:
I thought we agree on 406

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/88/commits/b90c26f82c8a10a4e8a8a02eeb438815d7991180

Reply:
I changed to 409